tap_hubspot/hubspot.py

```diff
@@ -18,8 +18,16 @@ class InvalidCredentials(Exception):
     pass
 
 
-def backoff_with_offset(backoff, offset=10):
-    return lambda: (n + offset for n in backoff)
+def giveup_http_codes(e: Exception):
```

Comment:
Could be simplified as

```python
def giveup_http_codes(e: Exception):
    return isinstance(e, requests.HTTPError) and e.response.status_code in {400, 404}
```

Reply:
added a few extra cases that makes this less practical